Logger/Person.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Logger
+{
+
+    //implemented implicitely so other entities could use it
+    public record class Person : IEntity
+
+    {
+        public Guid Id { get; init; }
+        public FullName FullName { get; init; }
```

Comment:
```suggestion
        public FullName FullName { get; }
```
This does not need an init setter on it.